lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -298,6 +315,8 @@ private static String normalize(String name) {
     for (char c : name.toCharArray()) {
       if (isValidChar(c)) {
         normalizedNameSb.append(c);
+      } else if (c == '-') {
+        normalizedNameSb.append("minus");
```

Comment:
Good.

Reply:
Note that @radeusgd agrees that group names and spec names *should* be restricted to valid Java identifiers. I will add this check to the Enso code that collects the benchmarks. There will be no more need for any name conversions.